### PR TITLE
product card

### PR DESCRIPTION
**Pull request by andrii-petruk**
- [DEMO LINK]( https://andrii-petruk.github.io/layout_product-cards/ )
- [TEST REPORT LINK]( https://andrii-petruk.github.io/layout_product-cards/report/html_report/ )

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  font-weight: 400;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
```
Comment (loralevitska) [inline src/style.css:14]: It is a good practice to make all colors variable

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 40px;
+}
+
+.card__description {
+  margin: 0 16px;
+  margin-bottom: 4px;
+  font-style: normal;
```
Comment (loralevitska) [inline src/style.css:28]: Remove redundant default properties. Fix all cases if present

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin: 0 16px 16px;
+  font-style: normal;
+  font-weight: 400;
```
Comment (loralevitska) [inline src/style.css:38]: Remove redundant default properties. Fix all cases if present

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,59 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p class="card__reviews">Reviews: 5</p>
+      </div>
+
+      <div class="price-container">
```
Comment (loralevitska) [inline src/index.html:40]: Fit it 
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Please deploy your solution, and before review check demo_link. If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review
image

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+  font-style: normal;
+
+  --color-blue: #00acdc;
+  --color-dark: #060b35;
+  --color-gray: #616070;
+  --color-white: #fff;
+
+}
+
+.card {
```
Comment (SanyaBratashchuk) [inline src/style.css:15]: move each BEM block to the separate file

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
```
Comment (SanyaBratashchuk) [inline src/style.css:21]: movr color to the variable

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 5 above)
Comment (andrii-petruk) [reply src/style.css:15]: An error occurs when moving blocks to separate files 
"C:\Users\Asusr\projects\layout_product-cards\src\style.css:4:1: Failed to install stars.css.
  2 | 
  3 | @import url(card.css);
> 4 | @import url(stars.css);
    | ^
  5 | 
  6 | html {"
Знімок екрана 2023-04-23 100908

**Pull request closed without merging**